html-forms/index.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>HTML Form</title>
+</head>
+<body>
+  <form action="">
+    <div>
```

Comment:
This should be a `label` element. Make sure you use a `for` attribute with a value that matches the `id` of the form control element this is labelling!

Reply:
So a `form` element should have a `label` element wrapped around it as well?